contracts/contracts/CAPE.sol

```diff
@@ -249,9 +249,11 @@ contract CAPE is RootStore, AssetRegistry, ReentrancyGuard {
 
     /// @notice Submit a new block with extra data to the CAPE contract.
     /// @param newBlock block to be processed by the CAPE contract
-    /// @param extraData extra data to be stored in calldata; this data is ignored by the contract function
-    // solhint-disable-next-line no-unused-vars
-    function submitCapeBlockWithMemos(CapeBlock memory newBlock, bytes calldata extraData) public {
+    /// @param {bytes} extraData data to be stored in calldata; this data is ignored by the contract function
+    function submitCapeBlockWithMemos(
+        CapeBlock memory newBlock,
+        bytes calldata /* extraData */
```

Comment:
Do we need this parameter now?

Reply:
Yes the memos are still provided via calldata. We can also change that but I'm not sure it's the best thing to do since it's just a workaround for not having another solution to send memos. Unlike the case of the CapeBlock, emitting the memos as an event does not provide additional robustness because the memos cannot be verified for correctness in the contract. Or in other words, a relayer could choose to send bad or no memos if they want to, even if we emit them as an event.